### ISSUE TITLE
Desktop app doesn't know about OS X Spaces

### ISSUE THREAD

**Issue by Sharpie** [cappuccino/issues#24]
Aweseome app- played around with the online version and just tried the stand-alone desktop model.  One issue that I have noticed with the desktop model is that it does not play nice with the multiple desktops provided by OS X spaces.

I can not:
- Grab the main window, drag it to the edge of the screen and have it switch to the next space.
- Grab the main window and use the "Switch between spaces" hotkeys ( i.e. Ctr+arrow key ) to move the app to a different desktop.
- Activate spaces (multiple desktop view) and drag the app window to another space.

Basically, if I want to move the Issues desktop app to another deskop I have to quit it, move to that desktop and then relaunch issues.

Overall though:

Awesome job!
